.eslintrc.js

```diff
@@ -6,7 +6,7 @@ const config = createConfig({
   cwd: __dirname,
   rules,
   overrides: [
-    ...overrides,
+    ...overrides.slice(1), // exclude shared React overrides (`react` is not installed at the root of the repo)
```

Comment:
Got an error because I turn on a rule from `eslint-plugin-react-hooks`, and this plugin is only enabled by Galex when it detects that `react` is installed.